### DISCUSSION TITLE
Question about estimating BA using KDM_calc

### DISCUSSION THREAD

**Discussion by mh231129** [dayoonkwon/BioAge#27] [Q&A] [answered]
Hello:

We are working on dataset of NHANES 15-18 cycles and trying to estimate BA using the KDM_calc. We use the same 9 biomarkers as those included in PhenoAge. We can train these markers in NHANES III, and project into NHANES IV. We notice that you clarified study design as survey and listed weights =1 in the coding of KDM_calc. We are wondering whether we should directly apply KDM_calc into our testing data NHANES 15-18, or we need to modify the coding of KDM_calc to reflect changes in weights. 

Thanks!

**Comment by dayoonkwon** [comment, marked as answer]
There's no need to modify the KDM_calc code.
